maxi-blocks/maxi-blocks
# Fixes #2851 - Loading proper SC fonts from google

# Description

Font loader wasn't ready to deal with changes on SC. An example: a creator does post-1 with a `Roboto` SC font-family for paragraph, and publishes it. Then, in post-2 changes the font-family for the paragraph to `Montserrat` and publishes it. If we check post-1 is going to stay on Roboto, but being a SC value should affect all pages.
Other issue found was that `getAllFonts` just get one font for every block, so it means it wasn't able to get different fonts for every responsive stage.

What has been done is create an alternative name for the elements which font-family relies on SC that is used on the server side to request for the SC value. Also has breakpoint support, but then I found is not necessary as Font Family is just shown on general responsive stage on the SC lol

Also implemented php linter standard on the code settings 👍 

PS: we need unit and e2e for this implementation, is too important and delicate 👍 

Fixes #2851 

# How Has This Been Tested?

To test it we've got 2 different situations: with local fonts activate/deactivate, so test next steps with both situations:
1. Create a post, and set different blocks, like text (with different headings), button or image (the ones that have text part)
2. Publish the post and check frontend
3. Create a new post, do the same than step 1
4. Change SC font-family for some things and then apply the SC.
5. Publish the post, check frontend and then check the frontend of the first post after reloading: it should get the new font 👍 

# Test checklist

**_ Front/Back Testing _**


**_ Pre-Code Testing _**


# Checklist
